pkg/framework/io.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+/**
+Variables because most Kubernetes resources access this attribute by reference
+*/
+
+var (
+	// ModeForCertificates unix file mode for certificates.
+	ModeForCertificates = int32(0640)
```

Comment:
not sure as well if create a io.go is a good idea, maybe centralize it on the class where the config map and secrets is being created?

Reply:
There's not only **one** place that make use of this. The `framework` package purpose is to offer meaningful features/constants to k8s API interaction, that's ok having one minor file now to handle this. Plus it's possible to import this package from any other package in the project.